packages/passport/sdk/src/zkEvm/walletHelpers.ts

```diff
@@ -119,31 +119,45 @@ export const getSignedTypedData = async (
   // @ts-ignore
   delete types.EIP712Domain;
 
-  // eslint-disable-next-line no-underscore-dangle
-  const digest = ethers.utils._TypedDataEncoder.hash(typedData.domain, types, typedData.message);
+  // Hash the EIP712 payload and generate the complete payload
+  const { _TypedDataEncoder: typedDataEncoder } = ethers.utils;
+  const digest = typedDataEncoder.hash(typedData.domain, types, typedData.message);
```

Comment:
Small preference would be renaming `digest` to `typedDataDigest` or `typedDataHash`.

And naming `completePayload` to `messageSubDigest`.

Might also help to include a link here to the contracts: https://github.com/immutable/wallet-contracts/blob/7824b5f24b2e0eb2dc465ecb5cd71f3984556b73/src/contracts/modules/commons/ModuleAuth.sol#L155

Reply:
Fixed ✅